### PR TITLE
Workload Details workbook

### PR DESCRIPTION
**Pull request by deagraw**
## PR Checklist

* [x] Added workload workbook templates for AKS and AKS Engine.
image
image
image

### CODE REVIEW

File: Workbooks/AKS/Workload Details AKS Engine/settings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Workload Details",
+    "author": "Microsoft",
+    "galleries": [
```
Comment (BLuEScioN) [inline Workbooks/AKS/Workload Details AKS Engine/settings.json:5]: How did you test that this workbook works? Without manually passing the parameters that would be passed in an AKS Engine environment, you can only do this in production. But of course, we wouldn't test in production. That's why there is a 'isPreview' property. Set that to true and this workbook will get deployed under a feature flag.

File: Workbooks/AKS/Workload Details AKS Engine/settings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        { "type": "container-insights", "resourceType": "Microsoft.ContainerService/managedClusters", "order": 600 },
+        { "type": "workbook", "resourceType": "Microsoft.ContainerService/managedClusters", "order": 600 }
```
Comment (BLuEScioN) [inline Workbooks/AKS/Workload Details AKS Engine/settings.json:7]: order has to be 700

File: Workbooks/AKS/Workload Details AKS/settings.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Workload Details",
+    "author": "Microsoft",
+    "galleries": [
+        { "type": "container-insights", "resourceType": "Microsoft.ContainerService/managedClusters", "order": 600 },
+        { "type": "workbook", "resourceType": "Microsoft.ContainerService/managedClusters", "order": 600 }
```
Comment (BLuEScioN) [inline Workbooks/AKS/Workload Details AKS/settings.json:7]: order has to be 700

File: Workbooks/AKS/Workload Details AKS Engine/settings.json (hunk 1 above)
Comment (deagraw) [reply Workbooks/AKS/Workload Details AKS Engine/settings.json:6]: Done

**Pull request merged**